synthtool/languages/node.py

```diff
@@ -94,6 +97,74 @@ def get_publish_token(package_name: str):
     return package_name.strip("@").replace("/", "-") + "-npm-token"
 
 
+def extract_clients(filePath: Path) -> List[str]:
+    """
+    parse the client name from index.ts file
+
+    Args:
+        filePath: the path of index.ts.
+    Returns:
+        Array of client name string extract from index.ts file.
+    """
+    content = ""
+
+    try:
+        with open(filePath, "r") as fh:
+            content = fh.read()
+    except FileNotFoundError:
+        err_msg = f"Failed to find {filePath}."
+        log.error(err_msg)
+        raise FileNotFoundError(err_msg)
```

Comment:
If we're going to raise a fatal error in the end, I don't see the advantage of catching it.  Please remove the try and except blocks.

Reply:
Got it. Remove it.